octobot/community/errors_upload/errors_uploader.py

```diff
@@ -0,0 +1,89 @@
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import asyncio
+import aiohttp
+
+import octobot_commons.logging
+
+
+class ErrorsUploader:
+    """
+    ErrorsUploader manages errors posts to the error url
+    """
+
+    def __init__(self, upload_url):
+        self.upload_url = upload_url
+        self.loop = None
+        self.upload_delay = 1
```

Comment:
I think we can largely increase this value. We don't need to upload errors as often. I think once a minute would be more than enough.

Reply:
there is a risk if we do that though: if the bot gets killed because users spot errors, we will never get them

Reply:
Do you really think 1 second is mandatory?